examples/bri-3/test/sriUseCase.e2e-spec.ts

```diff
@@ -334,3 +376,42 @@ async function fetchBpiAccount(bpiAccountId: string): Promise<any> {
 
   return JSON.parse(getBpiAccountResponse.text);
 }
+
+async function createEddsaPrivateKey(
```

Comment:
these 3 methods looks like are not e2e tests specific, do we use something similar in the rest of codebase, so we can move them in separate module and not duplicate?

Reply:
Although we don't have something similar, I added them to test utils, instead of the e2e tests.